fluster/test.py

```diff
@@ -95,13 +95,28 @@ def _test(self) -> None:
 
         if not self.reference:
             self.test_suite.test_vectors[self.test_vector.name].test_result = TestVectorResult.FAIL
-            if self.test_vector.result.lower() == result.lower():
+            if self.test_suite.name in {"MPEG2_VIDEO-422", "MPEG2_VIDEO-MAIN"}:
+                binary = "mpeg2decode"
```

Comment:
We have an abstraction for decoders, why would you implement the mpeg2 reference here instead of using the `Decoder` base class as it's done for the rest of reference decoders?